Defined "grandfathered in" (#341)
tatianamac/selfdefined: 11ty/definitions/grandfathered-in.md

---

> title: grandfathered in
> slug: 'grandfathered-in'

Reviewer: ```suggestion
title: grandfathering
slug: 'grandfathering'
```

speech: verb
defined: true
flag:
 level: avoid
 text: racist language

> excerpt: To permit to continue under expired rules

Reviewer: ```suggestion
excerpt: avoid. colloquially, to permit to continue under expired rules; racist origins of Grandfather Clause
```

alt_words:
  - legacy
  - historical
reading:
  - text: Massachusetts Court Won’t Use Term ‘Grandfathering,’ Citing Its Racist Origins
    href: https://www.nytimes.com/2020/08/03/us/racism-massachusetts-grandfathering.html
  - text: 10 common phrases that are actually racist AF.
    href: https://www.upworthy.com/10-common-phrases-that-are-actually-racist-af
  - text: "Words Matter: Why We Should Put an End to 'Grandfathering'"
    href: https://medium.com/@nriley/words-matter-why-we-should-put-an-end-to-grandfathering-8b19efe08b6a
  - text: The Racial History Of The 'Grandfather Clause'
    href: https://www.npr.org/sections/codeswitch/2013/10/21/239081586/the-racial-history-of-the-grandfather-clause
---

> To permit to continue under expired rules

Reviewer: ```suggestion
colloquially, to permit to continue under expired rules
```

## Issues

In the United States, ‘Grandfather Clause’ originally referred to provisions adopted by seven Southern states after the Civil War in an effort to disenfranchise African-American voters by requiring voters to pass literacy tests or meet other significant qualifications, while exempting from such requirements those who were descendants of men who were eligible to vote before 1867. This type of law ensured that illiterate whites were able to vote while restricting Black Americans from voting.  This practice of grandfather clauses did not officially end until 1965, when the Voting Rights Act was implemented.

## Impact

> The modern use of the phrase "grandfathered in" ignores its racist roots and normalizes a term that enfranchised white Americans and restricted Black Americans from voting.  Refraining to use this term is an acknowledgement of its history and a dedication to inclusivity by avoiding inciting images of racism out of context.

Reviewer: ```suggestion
The modern use of the phrases of "grandfathering" and "grandfathered in" ignores their racist roots and normalises a term that further enfranchised white Americans and restricted Black Americans from voting.  Refraining to use this term is an acknowledgement of its history and a dedication to inclusivity by avoiding inciting images of racism out of context.
```

## Usage Tip

> Use a neutral word like 'legacy' instead, or be more specific about the circumstances.  Rather than saying 'those customers were grandfathered in to the 2017 pricing schedule', rephrase it to 'those customers have retained the pricing schedule from 2017 due to the terms of service at the time' or 'those customers are on a legacy pricing plan from 2017.

Reviewer: I might suggest here splitting this into a couple of different examples.

> When possible, be more specific about the circumstances. Usually, we can acknowledge a specific timeframe or what benefits a group is receiving during which timelines.

> Substitute "grandfathered in" for more active language. "We allowed X people Y benefits since they were long-time customers (more than 10 years).

Also, the concept of grandfathering should be considered. Why are you providing benefits and/or exceptions to this group? Are you perpetuating the inequitable concepts that this word carries as a result? Removing just the word will not remove these inequities.

Author: Thank you! I split this out & added one additional comment regarding legality/contracts based on my own experience.  The term 'grandfathered in' was frequently used at a past employer of mine when referring to customers on older contracts which we were obligated to keep.  Happy to take it out if it doesn't add much to the definition.

I especially like your addition to reconsider why specific benefits/exceptions are allowed, and if they will continue inequities.
